### PR TITLE
Improve `ray.tune` usage example.

### PR DESCRIPTION
**Pull request by bwohlberg**
Improve `ray.tune` usage example. Also:
- Some minor improvements to `scico.ray.tune` wrapper.
- Extend tests for  `scico.ray.tune`.
- Add `ray.tune` usage example to examples index.

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -6,22 +6,6 @@ Usage Examples
...
-The additional requirements should be installed via pip, with the exception of ``astra-toolbox``,
-which should be installed via conda:
-
-   ::
-
-      conda install -c astra-toolbox astra-toolbox
-      pip install -r examples/examples_requirements.txt # Installs other example requirements
-
-The dependencies can also be installed individually as required.
-
```
Comment (smajee) [inline docs/source/examples.rst:24]: Did you remove the example related dependencies? I don't think these are there anywhere else.

File: scico/ray/tune.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2021 by SCICO Developers
+# Copyright (C) 2021-2022 by SCICO Developers
```
Comment (smajee) [inline scico/ray/tune.py:2]: For other files, should the end year be the current year or the year the file was last edited on?
In the long-term, should we manually update these or should we make a script (say in scico/misc) that updates these automatically.

**Review by smajee — APPROVED** [review]
Looks good other than the concern about removal of example dependencies.

File: docs/source/examples.rst (hunk 1 above)
Comment (bwohlberg) [reply docs/source/examples.rst:24]: This is going to be a continual pain until #195 is addressed. I'll fix manually for now.

File: scico/ray/tune.py (hunk 2 above)
Comment (bwohlberg) [reply scico/ray/tune.py:2]: My understanding is that the end year should correspond to the year of most recent edit. I think we can live with manual updates for now.

**Pull request merged**